plugins/DBSize/migrations/001_migrations.rb

```diff
@@ -0,0 +1,15 @@
+# DiskSize's migration file
+
+Sequel.migration do
+  transaction
+  change do
+    create_table(:db_size_measurements) do # Plugin's DB model
+      primary_key   :id
+      foreign_key   :node_resource
+      Time          :created, null: false
+      String        :node
```

Comment:
We should really not call these nodes. Nodes is more of a Cloud VM term. These should probably be called `databases` or `dbs`.